.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
can you add  gitignore with standard items like .idea , .vscode etc

Reply:
Definitely!